### DISCUSSION TITLE
要是有像vercel一样的平台支持就很爽了

### DISCUSSION THREAD

**Discussion by xmsz** [web-infra-dev/modern.js#3927] [General]
背景：
用过midway modernJS，『写』的环节已经都很完善了，但是部署和更多后端能力支持就蛋疼了...

部署：
得自己搞服务器，自己配置，自己部署
非常麻烦

后端能力：
比如数据库或者缓存，这些也要自己部署和提供

然后之前用nextjs，搭配vercel非常的方便

所以，希望有人也可以搞个类似vercel的平台，及时一开始就收费大家也能接受，当然前提是要稳定持续

然后，再说一下云开发的区别
之前还用过云开发，比如腾讯云的，其实也蛮方便的，就是限制很多，也不够『开放』，就是得按照他们的要求硬整，后面就放弃了
然后也搞过云函数部署之类的，但是也没有很方便，因为云函数适合简单需求

或者大家有什么好办法解决吗？

**Comment by chenjiahan** [comment]
哈哈，对于 Modern.js 团队来说，目前还没有计划推出类似 Vercel 的部署平台。

我们下一步的目标是提供完善的部署指南，使 Modern.js 应用可以被快速部署到主流的部署平台上，包括 Vercel，Netlify，阿里云等等，参考：https://github.com/web-infra-dev/modern.js/issues/3624
